### DISCUSSION TITLE
Sandbox: remafox deny file-read-data

### DISCUSSION THREAD

**Discussion by TopScrech** [FlineDev/RemafoX#103] [Help / FAQ]
I've unexpectedly begun to encounter an error when building the iOS target. Oddly enough, this issue doesn't occur for the macOS and tvOS targets. However, it's worth noting that there isn't any build script set up for these. The error message is as follows:

`Sandbox: remafox(15350) deny(1) file-read-data /Users/topscrech/<path>/Bisquit.Host.remafox
`

Initially, I suspected this problem might be related to the transition to Xcode 15, but the problem started appearing some time after that transition. Looking forward for reply

**Comment by Jeehut** [comment]
@TopScrech Thank you for reporting this. This sounds like you've set up the RemafoX build script and using Xcode 15 where for some reason the system disallows the build script from accessing files in your projects folder. There was no issue with accessing files within the projects folder from build scripts in the past, so maybe this is a new safety feature in Xcode 15. I haven't heard anything in the many sessions I watched last week about such a change. I also can't find anything related in the [Xcode 15 beta 1 release notes](https://developer.apple.com/documentation/xcode-release-notes/xcode-15-release-notes). Also if it worked in the first few builds and only happened at a later time, this might just be a bug in Xcode 15 beta 1 which you should file a feedback for.

I can't guarantee RemafoX works with beta Xcode versions. But to understand the issue better to come up with a workaround, I need more information. I just tested in my projects with Xcode 15 and didn't run in to this issue. To help you better, could you:

* Test building the same project in Xcode 14 (comment out new API code if necessary)? Do you get the same error?
* Post the env info in RemafoX's about page (Cmd + I) here?

**Reply by TopScrech** [reply]
Regrettably, I am unable to furnish further information as I no longer have access to previous commits. Moreover, I've made the decision to shift from RemafoX back to native localisation, given the substantial advancements brought about by Xcode 15. The improvements are so significant that RemafoX has become somewhat redundant. I sincerely appreciate your understanding and thank you for your previous responses!

**Reply by Jeehut** [reply]
@TopScrech I am aware of the situation. I‘m working on shifting the focus for the future of RemafoX so long-term subscribers continue to benefit. I’ll announce more details and link to it within the app before Xcode 15‘s final release. Thank you for giving RemafoX a try, I understand your decision and hope to win you back with future updates. 💪
